src/aws_encryption_sdk/keyring/base.py

```diff
@@ -0,0 +1,41 @@
+# Copyright 2017 Amazon.com, Inc. or its affiliates. All Rights Reserved.
+#
+# Licensed under the Apache License, Version 2.0 (the "License"). You
+# may not use this file except in compliance with the License. A copy of
+# the License is located at
+#
+# http://aws.amazon.com/apache2.0/
+#
+# or in the "license" file accompanying this file. This file is
+# distributed on an "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF
+# ANY KIND, either express or implied. See the License for the specific
+# language governing permissions and limitations under the License.
+"""Base class interface for Keyrings."""
+
+
+class Keyring(object):
+    """Parent interface for Keyring classes."""
```

Comment:
One last thing because I forgot about this until now..
```suggestion
    """Parent interface for Keyring classes.
    
        .. versionadded:: 1.5.0
    """
```